src/commands/run.ts

```diff
@@ -34,7 +34,9 @@ export async function run<T = any>(
   } = {},
 ): Promise<T[] | undefined> {
   const parsedSettings = await parseSettings(settings, shadow);
-  const sql = compilePlaceholders(parsedSettings, content, shadow);
+  const sql = await compileIncludes(
+    compilePlaceholders(parsedSettings, content, shadow),
+  );
```

Comment:
This is the wrong time to do this; the code _must_ be included into the committed migration, otherwise as the file evolves newer people who run the migrations from scratch will get a different result.